csrc/transform_view.cpp

```diff
@@ -773,4 +822,174 @@ TensorDomain* transformView(
   return createViewDomain(original_domain, view_analysis);
 }
 
+std::string AnalyzeViewResult::toString() const {
+  std::stringstream ss;
+  ss << "{ "
+     << "broadcast: " << broadcast_axes << ", squeeze: " << squeeze_axes
+     << ", transforms: ";
+  bool first_transform = true;
+  for (const auto& transform : transforms) {
+    if (!first_transform) {
+      ss << ", ";
+    }
+    ss << transform->toString();
+    first_transform = false;
+  }
+  ss << " }";
+  return ss.str();
+}
+
+bool AnalyzeViewResult::operator==(const AnalyzeViewResult& other) const {
```

Comment:
Note: This comparison is used when comparing `DynamicTransformConcretizationInfo`. It is mostly the same as the equality comparison of `AnalayzeViewConstraint`, but here the original and new constraints, i.e., `AnalyzeViewConstraint::original_constraint` and `AnalyzeViewConstraint::new_constraint`, are not used as I don't think it's necessary.

Reply:
Tracked here #217